packages/models/src/lib/global-options.ts

```diff
@@ -15,6 +15,11 @@ export const globalOptionsSchema = z.object({
   config: filePathSchema(
     "Path to config file in format `ts` or `mjs`. defaults to 'code-pushup.config.js'",
   ).default('code-pushup.config.js'),
+  onlyPlugins: z
```

Comment:
can we add the property to the CollectOptions and AutorunOptions type instead of global options please